backend/tests/conftest.py

```diff
@@ -146,3 +147,8 @@ def delete_all_tables_if_existent(_engine: sqlalchemy.engine.Engine) -> bool:
         session.commit()
 
     return True
+
+
+@pytest.fixture()
+def development_mode() -> bool:
+    return os.getenv("DEVELOPMENT_MODE", "").lower() in ("true", "1", "t")
```

Comment:
This is not a good idea, since this makes the tests dependent on external configuration.